scripts/docker/web/base/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# base image for elektra web, all other images build upon this (builds elektra with `yajl` and `kdb`)
+
+FROM ubuntu:16.04
+
+# elektra deps
+RUN apt-get update -y
+RUN apt-get install -y cmake git build-essential
+
+# elektra web deps
+RUN apt-get install -y libyajl-dev curl nodejs-legacy npm
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+
+# pull latest libelektra
+RUN mkdir /home/elektra
+WORKDIR /home/elektra
+RUN git clone https://github.com/omnidan/libelektra.git
```

Comment:
Don't you want to use ElektraInitiative's repo here?
you also want git clone --branch my_abc http://git.abc.net/git/abc.git and probably a shallow clone.

Reply:
you are right, I will do that now that the other PR is merged and rebuild the images

Reply:
I like to get .tar.gz archives from github. This removes the need to install git and it makes it very easy to insert a build-arg that represents the version I want to build.